dev-haskell/reflex/Manifest

```diff
@@ -1 +1,2 @@
 DIST reflex-0.6.3.tar.gz 145821 BLAKE2B 720a106815dee15ee0d2e02de2ab6b2e06e40eaafd01993ea7b9424d40e43c416b833648511bd7b7fb1cef3532bacd624d5dc619c36d83f8e4308cc975282763 SHA512 cd3114a90cb3bd028f7ba947743eb6dece68eb0857f42912abea3da77fd8b12af3499a15458828ff0186f771354f18efca7c30e19717f7043ddc06d184276230
+DIST reflex-0.8.0.0.tar.gz 141007 BLAKE2B 6219f63abd6ba347c0583f0a9c448e13c998a8414666bd655ed090ea875637dcb100fe2ed65bb1eb94284a6554098fbc3912601312d33a06fa4baa94f87d5a45 SHA512 da7fbf77a7afc000ab5f54f6cb6f83b0aad3740ea4f75ba2469076d72a1c1f6d3eda05633d2c78b75aa4c0ea6dc31298ab2ba5be50e25436bb6eeb91d187c950
```

Comment:
The reflex package has an "ecosystem" of associated packages. The issue i reference has a bunch of these "ecosystem" packages listed. Rather than open a new issue in each project, i elected to reference the preexisting issue.

What do you mean about the rebase spam?

Reply:
https://github.com/diagrams/diagrams-core/issues/112 pages and pages of updated from this PR.

Reply:
Oh yes, I did notice the spam. I was unable to find a way to remove it though :(

Reply:
After discussing it with the #github folks, it sounds like the only way to avoid this spam is to simply avoid referencing PRs/issues/what-have-you in commits unless you *know* that the commit will remain unchanged (i.e. you're pushing straight to master).

Therefore, in the future I will instead reference the relevant upstream issues/PRs here in a PR in `gentoo-haskell` OR an issue in `gentoo-haskell` OR wait until the commit is ready to go to master to add the reference.

I do think having the references to upstream bugs is helpful, but I also recognize that the spam is probably counterproductive.